incubator/mssql-linux/templates/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "mssql.fullname" . }}
+  labels:
+    app: {{ template "mssql.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "mssql.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "mssql.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: ACCEPT_EULA
+              value: "{{ .Values.acceptEula.value | upper }}"
+            - name: MSSQL_PID
+              value: "{{ .Values.edition.value }}"
+            - name: SA_PASSWORD
+              valueFrom:
+               secretKeyRef:
+                 name: {{ template "mssql.fullname" . }}-secret
+                 key: sapassword
+          ports:
+            - containerPort: 1433
```

Comment:
Give the port a name and reference it for configuring probes.

Reply:
Thanks.  I've updated the port with a name and referenced it in the probes.